### ISSUE TITLE
Not an issue just a question

### ISSUE THREAD

**Issue by AlbertEinsteinGlitchPoint** [sherreria/EponSimulator#1]
Hello

its not an issue but instead a question, can it be run on linux system, can i compile it on a linux server for testing?

can i emulate a OLT E-PON device? just need to have a E-PON OLT GBIC and on Cliente side a E-PON ONU GBIC to simulate testing?

**Comment by sherreria** [comment]
Yes, it can be run on a Linux system. And, yes, you can simulate a simple OLT E-PON device.
